TinkeringAudio

```diff
@@ -0,0 +1,188 @@
+#To play main so far type  testb()
+# TEST TEST TEST section that i test code in
+#zzzzzzzzzzzzzzzz section is the code that works and will be in main code
+#currently not working is stuff that is not working atm
+
+
```

Comment:
Follow the [PEP-8 conventions for blank lines](https://www.python.org/dev/peps/pep-0008/#blank-lines) in order to make the code neater.